subnets.tf

```diff
@@ -40,12 +40,22 @@ resource "aws_subnet" "private-b" {
     }
 }
 
-resource "aws_subnet" "data" {
+resource "aws_subnet" "data-a" {
```

Comment:
@denissstar could you keep the name of the subnet to be the same as previous to avoid any destroy from terraform please :)